walk-safe/src/Components/NavBar/_navbar.scss

```diff
@@ -2,20 +2,99 @@
   display: flex;
   flex-flow: row nowrap;
   justify-content: space-between;
-  align-items: center;
   width: 100%;
   height: 15%;
 }
 
 .welcome-msg {
   padding: 1em;
   align-self: flex-start;
-  font-size: 1.75em;
+  color:rgba(1, 15, 15, 0.712);
+  font-size: 2.5em;
   font-family: $roboto;
   font-style: italic;
+  letter-spacing: 4px;
 }
 
-.hamburger-container {
-  padding: 1em;
-  margin-right: 1em;
-}
\ No newline at end of file
+// BURGER
+
+.menu-item {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: ghostwhite;
+  font-style: $roboto;
+  font-size: 1.75rem;
+  margin: 2em auto 2em auto;
+
+  &:hover {
+    color: $light-blue;
+  }
+
+  &:active {
+    color: black;
+  }
+
+}
+
+/* Position and sizing of burger button */
+.bm-burger-button {
+  position: fixed;
+  width: 36px;
+  height: 30px;
+  right: 2em;
+  top: 3em;
+}
+
+/* Color/shape of burger icon bars */
+.bm-burger-bars {
+  background: #373a47;
+}
+
+/* Color/shape of burger icon bars on hover*/
+.bm-burger-bars-hover {
+  background: #d34c4c;
+}
+
+/* Position and sizing of clickable cross button */
+.bm-cross-button {
+  height: 40px;
+  width: 40px;
+}
+
+/* Color/shape of close button cross */
+.bm-cross {
+  background: #bdc3c7;
+}
+
+/*
+Sidebar wrapper styles
```

Comment:
Looking forward to pulling this down and messing with it!

Reply:
Please do!